compiler/crates/relay-typegen/src/visit.rs

```diff
@@ -588,6 +625,35 @@ fn import_relay_resolver_function_type(
         &PathBuf::from(resolver_metadata.import_path.lookup()),
     );
 
+    let context_import = if resolver_metadata.live {
+        match &typegen_context
```

Comment:
Optional: Might be cleaner to model this match as a `.map` since the Some case always returns a Some and the None case always returns None.